src/main/java/com/nexmo/quickstart/sms/ReceiveSMS.java

```diff
@@ -0,0 +1,26 @@
+package com.nexmo.quickstart.sms;
+
+import static com.nexmo.quickstart.Util.configureLogging;
+import static spark.Spark.*;
+
+public class ReceiveSMS {
+
+    public static void main(String[] args) throws Exception {
+        configureLogging();
+
+        port(8080);
+
+        get("/inbound", (req, res) -> {
+            for (String param : req.queryParams()) {
+                System.out.printf("%s: %s\n", param, req.queryParams(param));
+            }
+            return "OK";
+        });
+
+        post("/inbound", (req, res) -> {
```

Comment:
I'm guessing this is because the framework gives you a `get` and a `post` method but maybe not a method for both?